lib/binding/diag_kvdb_interface.c

```diff
@@ -154,3 +156,9 @@ diag_kvdb_get_cndb(struct hse_kvdb *handle, struct cndb **cndb)
 
     return 0UL;
 }
+
+struct mpool *
+diag_kvdb_get_mpool(struct hse_kvdb *handle)
+{
+    return ikvdb_mpool_get((struct ikvdb *)handle);
+}
```

Comment:
Hate to extend this diag API :(

Reply:
Why do you need this? There is a regular API to get the mpool right?

Reply:
No such API until you get to the `ikvdb` layer.